app/views/publisher_mailer/login_email.html.slim

```diff
@@ -1,14 +1,19 @@
 / Sent when user requests a login link via Publishers #create_auth_token
 - content_for(:title) do
-  h3= I18n.t("publisher_mailer.login_email.title")
+  h3= t("publisher_mailer.login_email.title")
 
-p= I18n.t("publisher_mailer.login_email.body_1")
-p= I18n.t("publisher_mailer.login_email.body_2")
-p= I18n.t("publisher_mailer.login_email.body_3")
+p.salutation= t("publisher_mailer.shared.salutation", name: @publisher.name)
 
-h4= "#{I18n.t("publisher_mailer.shared.private_access_link")}:"
-p= link_to(@private_reauth_url, @private_reauth_url)
+p= t("publisher_mailer.login_email.body_html")
+div.center= link_to(t("publisher_mailer.login_email.button"), @private_reauth_url, class: "btn-primary btn-login")
+p= t("publisher_mailer.shared.signature")
 
-p.hint
-  span.attribute= "#{I18n.t("publisher_mailer.shared.contact_help")}:"
-  = mail_to(Rails.application.secrets[:support_email])
+hr
+
+.hint
+  p
+    span= t("publisher_mailer.shared.button_not_working")
+    span= link_to(@private_reauth_url, @private_reauth_url, class: "fallback-link", onMouseOver: "this.style.color='#48494b'", onMouseOut: "this.style.color='#818589'")
```

Comment:
minor: i have a hunch that `onMouseOver` isn't supported in Gmail, however :hover is i think